src/style.css

```diff
@@ -0,0 +1,45 @@
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  margin: 0;
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star-active.svg");
+  background-size: auto;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(n+1) {
```

Comment:
```suggestion
.stars--0 .stars__star:nth-child(n+1),
.stars--1 .stars__star:nth-child(n+2),
.stars--2 .stars__star:nth-child(n+3),
.stars--3 .stars__star:nth-child(n+4),
.stars--4 .stars__star:nth-child(n+5),
.stars--5 .stars__star:nth-child(n+6) {
```
You can use commas to avoid code repeating and group selectors